src/components/field-types/MultiSelectFieldComponent.vue

```diff
@@ -3,36 +3,32 @@
     <div class="form-group">
       <label :for="field.id">{{ field.label }}</label>
 
-      <!--
-      /**
-        For creating options that do not exist:
-          - taggable = true
-          - pushTags = true
-          - createOption = Function
-      */
+      <div class="input-group">
 
-      /**
-        Filterable set to false because objects are only filtered on the label parameter
-          we want filtering on multiple parameters
-      */
-      -->
-      <v-select v-model="localValue"
-                class="form-control"
-                :class="{ 'is-invalid' : state && (state.$touched || state.$submitted) && state.$invalid}"
-                :options="options"
-                :onSearch="fetchOptions"
-                :filterable="false"
-                :inputId="field.id"
-                :name="field.id"
-                :required="isRequired"
-                :disabled="field.disabled"
-                :multiple="true">
+        <v-select v-model="localValue"
+                  class="form-control"
+                  :class="{ 'is-invalid' : state && (state.$touched || state.$submitted) && state.$invalid}"
+                  :options="options"
+                  :onSearch="fetchOptions"
+                  :filterable="false"
+                  :inputId="field.id"
+                  :name="field.id"
+                  :required="isRequired"
```

Comment:
Required validation triggers when pressing the '+' button. Even when selecting an original option, the red border does not go away